---
layout: about
title: about
permalink: /
subtitle: <a href='#'>Affiliations</a>. Address. Contacts. Motto. Etc.

profile:
  align: right
  image: prof_pic.jpg
  image_circular: false # crops the image to make it circular
  more_info: >
    <p>Senior Research Scientist</p>
    <p>Dark Blue Therapeutics</p>

selected_papers: true # includes a list of papers marked as "selected={true}"
social: true # includes social icons at the bottom of the page

announcements:
  enabled: true # includes a list of news items
  scrollable: true # adds a vertical scroll bar if there are more than 3 news items
  limit: 5 # leave blank to include all the news in the `_news` folder

latest_posts:
  enabled: true
  scrollable: true # adds a vertical scroll bar if there are more than 3 new posts items
  limit: 3 # leave blank to include all the blog posts
---

I'm Joe Harman, a senior research scientist (and bioinformatician) at Dark Blue Therapeutics. 

I have ~8 years experience in academic research, starting in the de Bruijn lab as a research assistant in 2015. In 2018 I started my DPhil in the Milne lab (Weatherall Institute, University of Oxford), and successfully defended my thesis (with commendation) in 2023. Following submission of my thesis, I worked as a Postdoctoral researcher in the de Bruijn lab for a short period, before joining Dark Blue Therapeutics as a research scientist.

Throughout my career, I have practiced a combination of wet lab and bioinformatics. In terms of computational biology I am experienced with analysing RNA-seq, ChIP-seq, ATAC-seq, proteomics, and their single-cell equivalents. I also have some experience analysing 3C data (micrococal capture-C). During my DPhil, my thesis was focused on synthesising new knowledge in combining different 'omics datasets and using these to infer gene regulatory networks (GRNs). Towards the end of my DPhil much of my work was focused on generating and analysing single-cell multiomics data and constructing GRNs from these datasets. 

On the wet lab side, I have specialised in molecular biology, cloning, and CRISPR gene editing. Molecular biology methods include library preparation (RNA-seq, ChIP-seq, CUT&Tag, ATAC-seq), qPCR, ddPCR, western blotting, immunofluoresence imaging. I am experienced with performing and designing various cloning methods, including gibson assembly. I have used CRISPR gene editing approaches to engineer mESCs and leukaemic cell lines to generate INDELs, in-frame HDR insertions, and enhancer-excisions.

At Dark Blue Therapeutics, I have continued to practice both wet lab work and bioinformatics. I have leveraged my molecular biology experience to answer key mechanistic questions for multiple projects in the company, and I have performed assay development for early projects, where assays have been taken on by a compound screening team. On the computational biology side, I have been involved in developing protocols for Dotmatics ELN to facilitate formal analysis and reporting of screening data (requring development of my SQL knowledge!). I have also, in collaboration with consultants, helped establish bioinformatics infrastructure including: AWS S3 storage, Nextflow pipelines in AWS batch, GitHub code documentations and workflows, data backup policies, external collaboration policies. 