---
layout: about
title: about
permalink: /
subtitle: <a href='#'>Affiliations</a>. Address. Contacts. Motto. Etc.

profile:
  align: right
  image: prof_pic.jpg
  image_circular: false # crops the image to make it circular
  more_info: >
    <p>Senior Research Scientist</p>
    <p>Dark Blue Therapeutics</p>

selected_papers: true # includes a list of papers marked as "selected={true}"
social: true # includes social icons at the bottom of the page

announcements:
  enabled: true # includes a list of news items
  scrollable: true # adds a vertical scroll bar if there are more than 3 news items
  limit: 5 # leave blank to include all the news in the `_news` folder

latest_posts:
  enabled: true
  scrollable: true # adds a vertical scroll bar if there are more than 3 new posts items
  limit: 3 # leave blank to include all the blog posts
---

I'm Joe Harman, a senior research scientist (and bioinformatician) at Dark Blue Therapeutics (DBT). 

I have ~8 years experience in academic research, starting in the de Bruijn lab as a research assistant in 2015. In 2018 I started my DPhil in the Milne lab (Weatherall Institute, University of Oxford), and graduated in 2023. I joined DBT as a research scientist in 2023.

My work bridges computational and molecular biology. I enjoy using large-scale â€˜omics data to uncover how genes are regulated in cancer, while also being able to leverage molecular techniques to test these mechanisms in the lab. 

My DPhil was focused on incorporating generating and analysing various 'omics data (RNA, ChIP, ATAC, proteomics) to infer gene regulatory networks (GRNs) in cancer (both cell line models and patient data) and embryonic haematopoietic ontogeny (mESC's, mouse data). I was then able to use various molecular biology techniques and CRISPR genome editing to validate the findings from these GRNs in the lab.


On the wet lab side, I have specialised in molecular biology, cloning, and CRISPR gene editing. Molecular biology methods include library preparation (RNA-seq, ChIP-seq, CUT&Tag, ATAC-seq), qPCR, ddPCR, western blotting. I have used CRISPR gene editing approaches to engineer mESCs and leukaemic cell lines to generate INDELs, in-frame HDR insertions, and enhancer-excisions.

At DBT, I have performed various molecular biology assays and bioinformatics analyses to support a lead project. My work on assay development has helped establish screening platforms now used in early drug discovery projects. I have also developed Dotmatics screening protocols (using SQL and database knowledge) to further support screening platforms. In collaboration with consultants, I have established bioinformatics infrastructure, such as AWS and Nextflow pipelines for data analysis. 

I have also developed Dotmatics screening analysis protocols, requiring SQL and database knowledge. In collaboration with consultants, I have also established bioinformatics infrastructure including: AWS S3 storage, Nextflow pipelines in AWS batch, GitHub code documentations and workflows. 